examples/pubsub/gcp-pubsub/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"github.com/NYTimes/gizmo/examples/nyt"
+	"github.com/NYTimes/gizmo/pubsub"
+	"github.com/sirupsen/logrus"
+	"github.com/NYTimes/gizmo/pubsub/gcp"
+	"context"
+	"fmt"
+)
+
+const (
+	projID = "my-project"
+	topic = "my-topic"
+	subscription = "my-subscription"
+)
+
+func main() {
+	ctx := context.Background()
+	pub, err := gcp.NewPublisher(ctx, projID, topic)
+	if err != nil {
+		pubsub.Log.WithFields(logrus.Fields{
+			"error": err,
+		}).Fatal("unable to init publisher")
+	}
+
+	publishMessage(pub, &nyt.SemanticConceptArticle{
+		Title:  "It's a Dog World",
+		Byline: "By David López",
+		Url:    "https://www.nytimes.com/2018/06/04/its-a-dog-world",
+	}, ctx)
+	publishMessage(pub, &nyt.SemanticConceptArticle{
+		Title:  "I'm a big big dog, in a big big world",
+		Byline: "By David López",
+		Url:    "https://www.nytimes.com/2018/06/05/big-big-dog-big-big-world",
+	}, ctx)
+
+	sub, err := gcp.NewSubscriber(ctx, projID, subscription)
+	if err != nil {
+		pubsub.Log.WithFields(logrus.Fields{
+			"error": err,
+		}).Fatal("unable to init subscriber")
+	}
+
+	pipe := sub.Start()
+	for i := 0; i < 2; i++ {
+		gotMsg := <-pipe
+		fmt.Println("Received message", string(gotMsg.Message()))
+		gotMsg.Done()
+	}
+	sub.Stop()
+}
+
+func publishMessage(pub pubsub.MultiPublisher, dogArticle *nyt.SemanticConceptArticle, ctx context.Context) {
```

Comment:
Mind moving the context to the first param? The general style is to pass it as the first parameter: https://github.com/golang/go/wiki/CodeReviewComments#contexts

Also, since we're using the `MultiPublisher` here, let's pass an array of articles here so we only have to call it once time above, thoughts?

Reply:
Sure, my fault! :)